contracts/contracts/core/PreconfManager.sol

```diff
@@ -200,7 +212,7 @@ contract PreconfManager is
      * @param decayStartTimeStamp The start time of the decay
      * @param decayEndTimeStamp The end time of the decay
      * @param bidSignature The signature of the bid
-     * @param sharedSecretKey The shared secret key
+     * @param zkProof The zk proof
```

Comment:
```suggestion
     * @param zkProof The zk proof is an array of c and z, representing the challenge and z = k - c*sk
```

Reply:
technically I guess that isn't the case, since you're wrapping a bunch of data into that array.